src/features/click-to-load.js

```diff
@@ -856,6 +870,7 @@ function unblockClickToLoadContent (message) {
 function runLogin (entity) {
     const action = entity === 'Youtube' ? 'block-ctl-yt' : 'block-ctl-fb'
     unblockClickToLoadContent({ entity, action, isLogin: true })
```

Comment:
Here, it's using `messaging.request('unblockClickToLoadContent', message)`, but you are not actually using the return value.

This can indicate an area to improve the design. For example, if it's truly a fire-and-forget message that you don't care about waiting for, then create a new message to serve that purpose (and use `.notify()` instead)

Or, it might be the case that you want to wait for the response, but don't need the value - in that case just use `await` on this and it might solve other synchronisation bugs too.

A bigger point to make here is that the distinction between `notify` and `request` is very deliberate - it's there to get you thinking deeply about the program flow whether or not things can/should be concurrent, serial or anything else.

Reply:
Thank you for the feedback.
I think that in this case, we can await the Platform response before firing the event to the surrogate. In my tests, it seems it was still working without calling this in sync, but I agree it might be a good practice to wait the rule to be unblocked before notifying the surrogate we're ready to login.
What do you think, @kzar?